elastic/datadog_checks/elastic/elastic.py

```diff
@@ -439,6 +444,61 @@ def _process_cat_allocation_data(self, admin_forwarder, version, base_tags):
                 desc = CAT_ALLOCATION_METRICS[metric]
                 self._process_metric(cat_allocation_dic, metric, *desc, tags=tags)
 
+    def _run_custom_queries(self, admin_forwarder, base_tags):
+        self.log.debug("Running custom queries")
+
+        custom_queries = self._config.custom_queries
+        for endpoints in custom_queries:
+            columns = endpoints.get('columns', [])
+            path = endpoints.get('path')
+            endpoint = endpoints.get('endpoint')
+            static_tags = endpoints.get('static_tags', [])
+            endpoint = self._join_url(endpoint, admin_forwarder)
+
+            data = self._get_data(endpoint)
+
+            # Search for any tags in columns first before processing metrics
+            dynamic_tags = self._process_tags(data, columns, path)
+
+            tags = base_tags + static_tags + dynamic_tags
+
+            for column in columns:
+                metric_type = column.get('type', 'gauge')
+
+                if metric_type == 'tag':
+                    continue
+
+                dd_name = column.get('dd_name')
+                es_name = column.get('es_name')
+                es_metric_path = path + es_name
+
+                if dd_name and es_name:
+                    self._process_metric(data, dd_name, metric_type, es_metric_path, tags=tags)
+
+    def _process_tags(self, data, columns, path):
+        dynamic_tags = []
+        for column in columns:
+            if column.get('type') == 'tag':
+                es_name = column.get('es_name')
+                dd_name = column.get('dd_name')
+                tag_path = path + es_name
+                value = data
+
+                for key in re.split(r'(?<!\\)\.', tag_path):
+                    if value is not None:
+                        value = value.get(key.replace('\\', ''))
+                    else:
+                        self.log.debug("Dynamic tag not found: %s -> %s", tag_path, dd_name)
```

Comment:
Can we make this log line different that the other one: https://github.com/DataDog/integrations-core/pull/10894/files#diff-c6a16fd0b579ee0555b3cf608278193ad35ebad8c6200757b1077e04ca95ab54R498